components/script/dom/globalscope.rs

```diff
@@ -263,6 +263,11 @@ impl GlobalScope {
         unreachable!();
     }
 
+    /// Get the origin URL for the current global object, shorthand for GlobalScope::current().get_url()
+    pub fn origin() -> ServoUrl {
```

Comment:
You're right. Won't save _that_ much typing anyway either. Can you help me with my feedback requests in the PR description?